query/control/controller_test.go

```diff
@@ -53,7 +54,7 @@ var (
 	bothConfigs          = map[string]control.Config{"unlimited": config, "limited": limitedConfig}
 	executorDependencies = []flux.Dependency{
 		influxdb.Dependencies{
-			FluxDeps: executetest.NewTestExecuteDependencies(),
+			FluxDeps: dependenciestest.Default(),
```

Comment:
`NewTestExecuteDependencies` in flux was changed, it used to be just a wrapper around `dependenciestest.Default()`. Keep the status quo here.